src/js/components/jobs/ctaButton.tsx

```diff
@@ -8,15 +8,22 @@ import { Job } from '@/store/jobs/reducer';
 import { CONSTANTS } from '@/store/plans/reducer';
 
 const { STATUS } = CONSTANTS;
+const btnClasses = 'slds-button slds-button_brand slds-p-vertical_xx-small';
```

Comment:
:heavy_plus_sign:  makes the render look cleaner; I'd love it if we use this pattern more for longer class lists.